README.md

```diff
@@ -22,14 +22,20 @@ AWS offers two services to manage secrets and parameters conveniently in your co
 
 ### Installing the AWS Provider and Config Provider (ASCP)
 
-#### Option 1: Using helm
+#### Option 1: Using eksctl
+
+```shell
+eksctl create addon --name aws-secrets-store-csi-driver-provider --cluster <your_cluster>
```

Comment:
Because the addon installs the driver and the provider this is not compatible with the requirements section above.